recipes/cigetcert/meta.yaml

```diff
@@ -0,0 +1,53 @@
+{% set name = "cigetcert" %}
+{% set version = "1.19" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: db7c3226552d23516aa16a767dfce8031362640fc7a94d87a0a6e90d5c8afb36
+
+build:
+  number: 0
```

Comment:
Can this be `noarch: python`?

Reply:
@xhochy, no, because it uses `scripts` in `setup.py`.

Reply:
Also, as it turns out, `pykerberos` isn't available on windows, so I need to skip that for now, and will work with upstream to try and support windows in the future.